db/schema.rb

```diff
@@ -21,9 +21,9 @@
   end
 
   create_table "copies", force: :cascade do |t|
-    t.string "Book_Id"
-    t.string "Borrower"
-    t.string "Due_Date"
+    t.string "book_id"
+    t.string "borrower"
+    t.string "due_date"
```

Comment:
I didn't notice this before, but you should probably change book_id's type to int and due_date's type to date. It's probably also worth adding in an explicit foreign key relationship between copies.book_id and books.id